mimic/test/behavior_tests.py

```diff
@@ -128,6 +131,9 @@ def make_behavior_tests(behavior_helper_factory):
 
     - providing invalid JSON will result in a 400 when creating the behavior.
 
+    - sequence behavior will rotate through the behaviors provided to it (if
+      :bool:`behavior_helper_factory.test_sequence` is true)
```

Comment:
One of the things that I really like about decorators as opposed to inheritance is that you don't have to do magic passing-implicit-parameters-by-setting-attributes-on-the-class, and you can just take an argument like a regular function.  I think this should be added as an argument to `behavior_tests_helper_class`.  I was going to say something about how to preserve compatibility with the old signature to allow for a graceful transition period, and then I remembered there are very few callers so far and Mimic is application code with no compatibility contract so hooray just go do it :).